extensions/positron-r/src/test/snapshots/indentation-snapshots.R

```diff
@@ -25,7 +25,7 @@ data |>
 
 # ->
 data |>
```

Comment:
I don't think these whitespace changes are expected, as this is part of a snapshot test case for indentation rules? @lionel- would know best.

Reply:
Fixed!